src/index.html

```diff
@@ -262,15 +274,21 @@ <h3 id="quiz-card__results">The Right answers is</h3>
         </details>
       </section>
 
+      <!--soll sich leeren .
+        wieder fokusieren .
+        aufs erste input .
+        anzahl der zeichen
+      soll begrenzt sein und runterzählen -->
+
       <section hidden data-js="pagecreate">
         <h2 class="underline">create your own question</h2>
         <h4>let your creativity flow...</h4>
-
+        <form data-js="form" onKeypress="zaehlen()" onKeydown="zaehlen()">
```

Comment:
`onKeypress="zaehlen()"` or `onKeydown="zaehlen()"` is not the way we add event listeners and not the way we name things (american english wording only).